packages/expo-sqlite/ios/NativeSession.swift

```diff
@@ -0,0 +1,13 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+
+final class NativeSession: SharedObject, Equatable {
+  var pointer: OpaquePointer?
+
+  // MARK: - Equatable
+
+  static func == (lhs: NativeSession, rhs: NativeSession) -> Bool {
+    return lhs.pointer == rhs.pointer
```

Comment:
If we are checking for identity equality, should we use `===`?

Reply:
that's to implement `Equatable` protocol and using `==` looks like common than `===`.